policyengine_canada/variables/gov/provinces/on/tax/income/credits/noec/northern_ontario_energy_credit.py

```diff
@@ -0,0 +1,59 @@
+from policyengine_canada.model_api import *
+
+
+class northern_ontario_energy_credit(Variable):
+    value_type = float
+    entity = Household
+    label = "Northern ontario energy credit"
+    unit = CAD
+    definition_period = YEAR
+
```

Comment:
Then you can remove from the eligibility check in the formula (also elsewhere)
```suggestion
    defined_for = "is_in_northern_ontario"

```